ledgerjs-fio/src/interactions/getPublicKey.ts

```diff
@@ -0,0 +1,61 @@
+import { PublicKey } from "../../dist/src/fio"
+import {ValidBIP32Path, WIF_PUBLIC_KEY_LENGTH} from "../types/internal"
+import {PUBLIC_KEY_LENGTH} from "../types/internal"
+import type {Version} from "../types/public"
+import {assert} from "../utils/assert"
+import {chunkBy} from "../utils/ioHelpers"
+import {path_to_buf} from "../utils/serialize"
+import {INS} from "./common/ins"
+import type {Interaction, SendParams} from "./common/types"
+import {ensureLedgerAppVersionCompatible} from "./getVersion"
+import {GetPublicKeyResponse} from "../fio"
+
+const send = (params: {
+    p1: number,
+    p2: number,
+    data: Buffer,
+    expectedResponseLength?: number
+}): SendParams => ({ins: INS.GET_EXT_PUBLIC_KEY, ...params})
+
+
+export function* getPublicKey(
+    version: Version,
+    path: ValidBIP32Path,
+    show_or_not: boolean
+): Interaction<GetPublicKeyResponse> {
+    ensureLedgerAppVersionCompatible(version)
+
+    const enum P1 {
+        SHOW = 0x01,
+        DO_NOT_SHOW = 0x02,
+    }
+
+    const enum P2 {
+        UNUSED = 0x00,
+    }
```

Comment:
I think these should be moved outside of the function.
```suggestion

const enum P1 {
    SHOW = 0x01,
    DO_NOT_SHOW = 0x02,
}

const enum P2 {
    UNUSED = 0x00,
}

export function* getPublicKey(
    version: Version,
    path: ValidBIP32Path,
    show_or_not: boolean
): Interaction<GetPublicKeyResponse> {
    ensureLedgerAppVersionCompatible(version)
```